### PR TITLE
Logical operators

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please read the description of article.md and solution.md of task 8 "if question".
Please review and comment / commit if anything is wrong

### CODE REVIEW

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 1]
```diff
@@ -1,109 +1,109 @@
...
+چهار عملگر منطقی در جاوااسکریپت وجود دارد: `||` (OR)، `&&` (AND)، `!` (NOT)، `??` (Nullish Coalescing). اینجا ما سه تای اول را پوشش می دهیم، عملگر `??` در مقاله بعدی است.
 
-Although they are called "logical", they can be applied to values of any type, not only boolean. Their result can also be of any type.
+با اینکه آنها "منطقی" نام برده می شوند، می توانند روی هر نوع مقداری اعمال شوند، نه فقط نوع boolean. نتیجه آنها هم می تواند از هر نوعی باشد.
 
-Let's see the details.
+بیایید جزییات را ببینیم.
 
-## || (OR)
+## || (OR یا)
```
Comment (mahdyar) [inline 1-js/02-first-steps/11-logical-operators/article.md:9]: There is no need for Persian translation.

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 2]
```diff
@@ -1,109 +1,109 @@
...
 
 ```js run
-alert( true || true );   // true
-alert( false || true );  // true
-alert( true || false );  // true
-alert( false || false ); // false
+alert( true || true );   // true درست
+alert( false || true );  // true درست
+alert( true || false );  // true درست
+alert( false || false ); // false اشتباه
```
Comment (mahdyar) [inline 1-js/02-first-steps/11-logical-operators/article.md:27]: Use either `true` or `درست`, not both of them.
Use either `false` or `اشتباه`, not both of them.

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 3]
```diff
@@ -115,175 +115,175 @@ This leads to some interesting usage compared to a "pure, classical, boolean-onl
...
+یک راه کلامی تر برای انجام کار مشابه وجود دارد -- تابع درون ساخت `Boolean`:
 
 ```js run
-alert( Boolean("non-empty string") ); // true
+alert( Boolean("string که خالی نیست") ); // true
 alert( Boolean(null) ); // false
 ```
 
-The precedence of NOT `!` is the highest of all logical operators, so it always executes first, before `&&` or `||`.
+اولویت NOT `!` بالاترین اولویت بین عملگر های منطقی دارد، بنابراین همیشه اول اجرا می شود، قبل از `&&` یا `||`.
```
Comment (mahdyar) [inline 1-js/02-first-steps/11-logical-operators/article.md:289]: The sentence readability could be better:
``عملگر NOT `!` بالاترین اولویت را بین عملگرهای منطقی دارد،``

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 4]
```diff
@@ -1,109 +1,109 @@
...
-We can pass more conditions:
+ما می توانیم شرط های بیشتری قرار بدهیم:
 
 ```js run
 let hour = 12;
 let isWeekend = true;
 
 if (hour < 10 || hour > 18 || isWeekend) {
-  alert( 'The office is closed.' ); // it is the weekend
+  alert( 'The office is closed.' ); // آخر هفته است
```
Comment (mahdyar) [inline 1-js/02-first-steps/11-logical-operators/article.md:63]: `The office is closed.` should also be translated.

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you for your review, @mahdyar

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**